docs-svelte-kit/build-system/build.mts

```diff
@@ -1,14 +1,16 @@
 import esbuild from "esbuild"
 import path from "path"
 import fs from "fs"
+import { fileURLToPath } from "url"
 // const babelCore = require("@babel/core")
 // const t = require("@babel/types")
 
-const dirname = path.dirname(
+const dirname = fileURLToPath(
   new URL(
```

Comment:
https://stackoverflow.com/questions/46745014/alternative-for-dirname-in-node-js-when-using-es6-modules

no need to use URL

Reply:
Hmm. I used `new URL('.', import.meta.url)` to get the directory, but your suggested method seems to be more common 👍